motioneye/handlers.py

```diff
@@ -1441,13 +1442,12 @@ def download(self, camera_id, filename):
         camera_config = config.get_camera(camera_id)
         if utils.is_local_motion_camera(camera_config):
             content = mediafiles.get_media_content(camera_config, filename, 'movie')
-            
+
```

Comment:
There's a lot of space removal changes like this one. They should not be part of this pull request.